corehq/apps/locations/static/locations/ko/location.js

```diff
@@ -0,0 +1,33 @@
+// for product and user per location selection
+$(function () {
+    $("#id_users-selected_ids").width(800).height(400).multiselect();
+    $("#id_products-selected_ids").width(800).height(400).multiselect();
+});
+
+var insert_new_user = function(user) {
+    $('#id_users-selected_ids').multiselect('addSelected', user.user_id, user.text);
+};
+
+$('#new_user').on('show', function() {
+
+    var url = NEW_MOBILE_WORKER_URL + "?location_id=" + LOCATION_ID;
+    var form_node = $("#new_user form");
+
+    $.get(url, function(data) {
```

Comment:
why is this being done as a separate request instead of on page load? it looks like LOCATION_ID is the only dependency and like that's known on initial page load.